dom.bs

```diff
@@ -894,8 +894,8 @@ for historical reasons.
 <pre class=idl>
 [Constructor, Exposed=(Window,Worker)]
 interface EventTarget {
-  void addEventListener(DOMString type, EventListener? callback, optional (AddEventListenerOptions or boolean) options);
-  void removeEventListener(DOMString type, EventListener? callback, optional (EventListenerOptions or boolean) options);
+  void addEventListener(DOMString type, EventListener? callback, optional (AddEventListenerOptions or boolean) options = false);
```

Comment:
Why make this change?

IIRC we discussed this on my original PR.  I could dig up the history but perhaps the argument against it was that it was observably identical to not specifying any default and instead relying on the dictionary defaults.  If behavior is indeed identical with or without this change, then I'd prefer we leave it out - the meaning when named `options` seems more explicit.  Also since we're now doing spec/impl IDL diffing, we'd want to get all the implementations to update which seems silly if there's no behavior difference whatsoever.